src/test/java/automationpractice/com/pages/WishlistPage.java

```diff
@@ -0,0 +1,65 @@
+package automationpractice.com.pages;
+
+import net.serenitybdd.core.pages.PageObject;
+import net.serenitybdd.core.pages.WebElementFacade;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class WishlistPage extends PageObject {
+
+    private static final String WISHLIST_NAME_PATTERN = "//*[contains(text(),'%s')]";
+    private final String DELETE_MY_WISHLIST_XPATH_PATTERN = "//*[contains(text(),'%s')]//ancestor::*[contains(@id,'wishlist_')]//*[contains(@class,'wishlist_delete')]/*[@class='icon']";
+
+    @FindBy(xpath = "//input[@id='name']")
+    private WebElementFacade wishlistName;
+    @FindBy(id = "submitWishlist")
+    private WebElementFacade submitButton;
+    @FindBy(xpath = "//*[contains(@class,'table-bordered')]//*[contains(@id, 'wishlist')]/td[1]")
+    private List<WebElementFacade> wishlistsNameList;
+    @FindBy(className = "wishlist_delete")
+    private WebElementFacade deleteWishlistButton;
+    @FindBy(xpath = "//*[@class='row wlp_bought_list']//*[@class='product-name']")
+    private List<WebElementFacade> productsList;
+
```

Comment:
Powinineś uzyć w tym xpath contains(@class, '')

Reply:
Jesli zmieni się kolejność klas 'row' i 'wlp_bought_list' to to przestanie działać. Wybierz jedna z tych klas tylko, lub jesli potrzebujesz obu to '//*[contains(@class,'row') and contains(@class, 'wlp_bought_list')]'